build/common.go

```diff
@@ -227,6 +228,13 @@ func Clean() error {
 	return nil
 }
 
+// E2EProxy starts the fixture proxy in append mode on port 9999.
+func E2EProxy() error {
```

Comment:
i'm having trouble thinking of a descriptive name. does `mage e2eproxy` seem ok?